### PR TITLE
Feature/responsive design

### PR DESCRIPTION
**Pull request by philalewis**
What (if any) features are you implementing?
- Finishing touches on styling
- Remove unused variables
- Formatting code for legibility
- Remove Stats page

What (if anything) did you refactor?
- Styling
- ErrorModal

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- Please review and merge

Any other comments, questions, or concerns?
- no

### CODE REVIEW

File: src/Components/ErrorModal.js [hunk 1]
```diff
@@ -3,20 +3,25 @@ import { Errors } from '../Contexts/ErrorsProvider'
...
-        <p>{errorMessage}</p>
-        <button className='modal-btn' onClick={() => setErrorMessage({error: null})}>OK</button>
-      </article>
-    </div>
-  )} else {
-    return(
-      <></>
-    )
-  }
+
```
Comment (oacortez) [inline src/Components/ErrorModal.js:8]: Nice use of conditional rendering, seems cleaner in my opinion!

File: src/Components/Home.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { useContext } from 'react'
-import { LeagueMembers, LeagueMembersProvider } from '../Contexts/LeagueMembersProvider'
```
Comment (GraceGardner) [inline src/Components/Home.js:2]: Nice noticing all of these un used things.

File: src/App.js [hunk 3]
```diff
@@ -35,7 +33,6 @@ const App = () => {
                   <Routes>
                     <Route exact path='/' element={<Home />} />
                     <Route exact path='/manage/' element={<Manage />} />
-                    <Route exact path='/stats/' element={<Stats />} />
```
Comment (oacortez) [inline src/App.js:38]: Thank you for removing stats component. This will be a future iteration.

File: src/Components/SingleCourse.js [hunk 4]
```diff
@@ -48,25 +46,49 @@ const SingleCourse = () => {
         <h2 className="course-name">{course.name}</h2>
         <p className="course-location">{course.city}, {course.state}</p>
       </div>
-      {course.rating > 0 && <p className="course-rating"><span className="bold">Rating:</span> {course.rating}/5</p>}
-      <p><span className="bold">Number of Holes:</span> {course.holes}</p>
+      {
```
Comment (GraceGardner) [inline src/Components/SingleCourse.js:49]: Thanks for re-formatting all of these

File: src/Styles/Navbar.scss [hunk 5]
```diff
@@ -5,13 +5,15 @@ nav {
   @include nav-shadow();
   background-color: $dark-navy;
   color: $white;
-  padding: 5%;
+  padding: 2% 4% 2% 4%;
```
Comment (GraceGardner) [inline src/Styles/Navbar.scss:8]: this could be shortened to 2% 4%

**Review by GraceGardner — APPROVED** [review]
Nice code clean up and styling

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch Phil! I see a lot of good adjustment on CSS and also in our JavaScript files.

**Pull request merged**